blivet/devicetree.py

```diff
@@ -862,7 +862,9 @@ def hide(self, device):
         if not device.exists:
             return
 
-        self._remove_device(device, force=True, modparent=False)
+        # XXX set modparent=True when hiding a VG to unset vg_name from PVs
+        modparent = device.type == "lvmvg"
+        self._remove_device(device, force=True, modparent=modparent)
```

Comment:
This chunk makes me nervous, both because I fear it may break things when users repeatedly hide/unhide disks and because I don't understand why we need to unset vg_name in the pvs.

Reply:
> it may break things when users repeatedly hide/unhide disks

`add_hook` for `LVMVolumeGroupDevice` sets the `vg_name` on all PVs so it shouldn't be a problem. But it definitely can cause some problems in the future.

> I don't understand why we need to unset vg_name in the pvs

Becase PV format `status` is `True` when `vg_name` is set and we don't allow removing "active" formats (`DeviceFormat._pre_destroy`) raises an exception.